models/api.models.js

```diff
@@ -98,10 +98,40 @@ const insertComment = (article_id, username, body) => {
     });
 };
 
+
+const updateArticleVotes = (article_id, inc_votes) => {
+   const numberArticleId = +article_id;
+ 
+   if (isNaN(numberArticleId)) {
+     return Promise.reject({ status: 400, msg: "Invalid article ID format" });
+   }
+ 
+   if (typeof inc_votes !== 'number') {
+     return Promise.reject({ status: 400, msg: "Invalid vote increment format" });
```

Comment:
This should also throw a PSQL error, so can be dealt with in your error handling middleware.